src/index.njk

```diff
@@ -46,25 +46,25 @@ tags: index-page # Note: This adds a custom class to layout.njk
       <div class="col-xl-3 col-md-6 d-flex align-items-stretch mt-4 mt-md-0" data-aos="zoom-in" data-aos-delay="200">
         <div class="icon-box">
           <div class="icon"><i class="bx bx-lock"></i></div>
-          <h4><a href="https://docs.kuadrant.io/latest/kuadrant-operator/doc/reference/tlspolicy/">Secure: TLSPolicy</a></h4>
+          <h4><a href="https://docs.kuadrant.io/latest/kuadrant-operator/doc/tls/">Secure: TLSPolicy</a></h4>
```

Comment:
Possibly the reason why it pointed to the API references before was because the DNSPolicy does not have an Overview page in the docs. IMO though the overview pages are usually richer to explain the concept for one who wants to learn about these policies for the first time.